### PR TITLE
Add BrowserInfo model to expose browser window and navigator APIs

### PR DESCRIPTION
**Pull request by philippjfr**
Provides various pieces information about the browser.

- `dark_mode`: Whether the user prefers dark mode.
- `device_pixel_ratio`: Provides the ratio of the resolution in physical pixels to the resolution in CSS pixels for the current display device.
- `language`: The preferred language of the user, usually the language of the browser UI.
- `timezone`: The timezone configured as the local timezone of the user.
- `timezone_offset`: The time offset from UTC in minutes.
- `webdriver`: Indicates whether the user agent is controlled by automation.

### CODE REVIEW

**Comment by MarcSkovMadsen** [comment]
Just for info. Dash just added a Geolocation component.

File: panel/config.py [hunk 1]
```diff
@@ -115,6 +116,9 @@ class _config(_base_config):
     autoreload = param.Boolean(default=False, doc="""
         Whether to autoreload server when script changes.""")
 
+    browser_info = param.Boolean(default=True, doc="""
```
Comment (Hoxbro) [inline panel/config.py:119]: I think it would be better to be off by default.

Comment (philippjfr) [reply panel/config.py:119]: Can see that argument but can you explain your reasoning? One of the main reasons we want to use it is for things like the device pixel ratio to determine the resolution of datashader plots and hardly anyone will know to enable this.

Comment (Hoxbro) [reply panel/config.py:119]: The reason for the suggestion was that there was no reason in this PR why it should be on by default. 

You gave me two reasons.

**Pull request merged**